src/backend/utils/load/ag_load_labels.c

```diff
@@ -199,15 +202,18 @@ int create_labels_from_csv_file(char *file_path,
 
 
     while ((bytes_read=fread(buf, 1, 1024, fp)) > 0) {
-        if (csv_parse(&p, buf, bytes_read, vertex_field_cb, vertex_row_cb, &cr) != bytes_read) {
-            ereport(ERROR, (errmsg("Error while parsing file: %s\n", csv_strerror(csv_error(&p)))));
+        if (csv_parse(&p, buf, bytes_read, vertex_field_cb,
+                      vertex_row_cb, &cr) != bytes_read) {
```

Comment:
Brace needs to be on its own line -

some_function_or_command(blah blah,...)
{